### DISCUSSION TITLE
Latest Wordpress plugin update - 'Room is not active'

### DISCUSSION THREAD

**Discussion by bhantegv** [mynaparrot/plugNmeet-server#481] [Q&A] [answered]
Hello.
I updated plugin on wordpress. I then tried to access Rooms and received 'Room not active' message.
I have four rooms enabled. None are working now.
Thank you.

**Comment by jibon57** [comment, marked as answer]
PNM need to upgrade to V1.6.0

**Reply by bhantegv** [reply]
Ok. Thank you.

**Reply by bhantegv** [reply]
I tried sending an email. What are your fees? I would like someone to take a look at our installation.
Thank you.

**Reply by jibon57** [reply]
You can send email jibon[@]mynaparrot.com

**Reply by bhantegv** [reply]
Very good. Email has been sent.
